creator-node/src/middlewares.js

```diff
@@ -760,7 +760,7 @@ async function getReplicaSetSpIDs({
       )
 
       try {
-        if (config.get('entityManagerReplicaSetEnabled')) {
+        try {
           // will throw error if blocknumber not found
           const encodedUserId = libs.Utils.encodeHashId(userId)
           const spResponse = await libs.discoveryProvider.getUserReplicaSet({
```

Comment:
We still want to wait to try and only call the `getUserReplicaSet` method in the release after this code freeze

Reply:
In the next release, we only want to be querying discovery for the replica set actually, there should be no fallback for reads